ci-scripts/yocto.groovy

```diff
@@ -77,6 +77,9 @@ void buildImageAndSDK(String yoctoDir, String imageName, boolean dev=true) {
         if (dev) {
             vagrant("/vagrant/cookbook/yocto/build-images.sh ${yoctoDir} ${imageName}-dev")
         }
+        if (update) {
+            vagrant("/vagrant/cookbook/yocto/build-images.sh ${yoctoDir} ${imageName}-update")
+        }
```

Comment:
Actually, this is a bit weird imho... what happens if you want to build an update to a -dev image? Is that possible?

Reply:
Correct, actually this is possible https://github.com/FlorentRevest/meta-pelux/tree/generic-swupdate-artifacts/recipes-extended/images
I could add an if (dev) then build the image-dev-update artifcat as well

Reply:
I guess you need to take it into account somehow at least :)